clients/src/main/resources/common/message/README.md

```diff
@@ -79,6 +79,8 @@ There are several primitive field types available.
 
 * "int64": a 64-bit integer.
 
+* "double": is a double-precision floating point number (IEEE 754).
```

Comment:
Should it be `float64` to be consistent with `int64`?

Reply:
I went with what proto uses. I have no preference and am fine with either.

Reply:
Interesting that they choose to include the size only for the integers. Since there has been a bit of an explosion of float types recently due to Machine Learning, I think `float64` is a bit more future proof. What do you think? Also, @hachikuji, do you have any thoughts?

Reply:
Updated to float64.